diff --git a/index.js b/index.js
index 1f6e87cc91e4dadd4d3e9d2338167225897c8846..41e2b2a2c80774fd462c65d1222f5f2b14cab987 100644
--- a/index.js
+++ b/index.js
@@ -16,6 +16,7 @@ var express = require('express');
 var compression = require('compression');
 var morgan = require('morgan');
 var express$1 = require('@remix-run/express');
+var path = require('path');
 
 function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }
 
@@ -27,12 +28,14 @@ function createApp(buildPath, mode = "production") {
   let app = express__default["default"]();
   app.disable("x-powered-by");
   app.use(compression__default["default"]());
-  app.use("/build", express__default["default"].static("public/build", {
+  const publicPath = path.resolve(buildPath, "..", "..", "public");
+  const publicBuildPath = path.resolve(publicPath, "build");
+  app.use("/build", express__default["default"].static(publicBuildPath, {
     immutable: true,
     maxAge: "1y"
   }));
-  app.use(express__default["default"].static("public", {
-    maxAge: "1h"
+  app.use(express__default["default"].static(publicPath, {
+    maxAge: "1h",
   }));
   app.use(morgan__default["default"]("tiny"));
   app.all("*", mode === "production" ? express$1.createRequestHandler({
@@ -41,7 +44,8 @@ function createApp(buildPath, mode = "production") {
   }) : (req, res, next) => {
     // require cache is purged in @remix-run/dev where the file watcher is
     let build = require(buildPath);
-
+    res.set("cross-origin-opener-policy", "same-origin");
+    res.set("cross-origin-embedder-policy", "require-corp");
     return express$1.createRequestHandler({
       build,
       mode